dev/reference/configuration/tikv-server/configuration-file.md

```diff
@@ -324,6 +324,16 @@ Configuration items related to Raftstore
 + Default value: `3s`
 + Minimum value: `0`
 
+### `raftstore.hibernate-regions` (**Experimental**)
+
++ Enables or disables Hibernate Region. When enabled, a Region idle for a long time is automatically set as hibernated. This reduces the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle Regions. You can use `raftstore.peer-stale-state-check-interval` to modify the the heartbeat interval between the leader and followers of hibernated Regions.
++ Default value: false
+
+### `raftstore.peer-stale-state-check-interval`
```

Comment:
Please check whether we need `(**Experimental**)` here.

Reply:
This is a dependent option, which depends on `raftstore.hibernate-regions` which is already indicated as Experimental.   Maybe we can add a note for this one, like "This option is only valid when  Hibernate Region is enabled.

Reply:
@dcalvin Good idea. Please help add some description.

Reply:
It's not true. It's an independent option, which works without hibernate-regions.